package.json

```diff
@@ -9,13 +9,20 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/amplitude/storybook-addon-amplitude"
+    "url": "git+https://github.com/amplitude/storybook-addon-amplitude.git"
   },
-  "author": "Mae Capozzi <mae.capozzi@amplitude.com>",
+  "type": "module",
+  "author": "Amplitude Inc",
   "license": "MIT",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/ts/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/ts/index.d.ts"
+    },
+    "./manager": "./dist/manager.js",
```

Comment:
@jimmynotjim  manager path it's not correct, it needs to be pointing to the ESM file (that is what Storybook expects):

"./manager": "./dist/**esm**/manager.js",

Reply:
Sorry, is that 👍  an "everything looks good" or an "I'll check it out when I can"? Just want to make sure I'm not holding you up any more than I already have getting these changes published.